data/human/free worlds side plots.txt

```diff
@@ -11,6 +11,88 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+
+
+mission "FW Convoy to Zug"
+	name "Convoy to Zug"
+	description "Escort these Freighters to Zug for an important project. Expect pirate raids."
+	source
+		government "Free Worlds"
+		near Zubeneschamali 2 4
+	destination "Zug"
+	to offer
+		has "event: war begins"
+		has "FW Syndicate Diplomacy 1D: done"
+		not "fw bloodsea done"
+	on offer
+		conversation
+			`While returning from a walk around the spaceport town of <origin>, you see the quartermaster of the spaceport inspecting the <ship>. He notices your arrival and comes up to greet you.`
+			`	"You're Captain <first> <last>, yes? We're looking for a good ship that can escort these freighters to <planet>." He points towards five Freighters in Free Worlds colors on the landing pads opposite to your ship, some of which have considerable hull damage and are actively being repaired.`
+			`	"Their last escort was just taken out by pirates, and we need someone who is experienced enough to fend off or evade them. You must have a reputation, since some of our militia captains recognized you. This is for quite a significant project on <planet>, so you will be paid <payment>."`
+			choice
+				`	"What is the project? It seems to be a big enterprise."`
+				`	"That's a fair few Freighters, should I be expecting any more pirates?"`
+					goto pirates
+				`	"Sorry, I'm not planning on going that way."`
+					decline
+			`	"The Freighters are carrying construction materials and specialized shipbuilding equipment for the extension of Zug's New Port, which has been a long time coming. It would greatly help the Free Worlds' efforts. I suppose that is why there have been pirates trying to pounce on these ships; the equipment would be a rich prize for them."`
+				goto meet
+			label pirates
+			`	"Yes, I would expect more pirate contact, because the movement of so many Freighters would be a tempting target for them, and moreover, they are carrying valuable equipment for warship manufacturing. I'd certainly recommend avoiding combat when possible. In these trying times, the loss of these ships would be... not ideal."`
+			label meet
+			`	"Well, will you help us escort these Freighters?"`
+			choice
+				`	"Of course, I'll get these Freighters to <planet> safely."`
+				`	"No, I won't be able to, sorry."`
+					decline
+			`	"Wonderful! I expect that these ships will benefit the Free Worlds immensely."`
+			`	With that, he leaves you to meet with the captains of the Freighters.`
+				accept
+	npc accompany save
+		government "Free Worlds"
+		personality escort timid
+		ship "Freighter" "F.S. Pohutukawa"
+		ship "Freighter" "F.S. Totara"
+		ship "Freighter" "F.S. Kauri"
+		ship "Freighter" "F.S. Rimu"
+		ship "Freighter" "F.S. Kanuka"
```

Comment:
I wonder if it might make more sense to use Haulers. I know FW Escort uses Freighters as well, though. Or maybe FW Escort should use Haulers, and this should use Freighters because of the Syndicate diplomacy stuff.

Reply:
They were originally Haulers III. The only problem was that they kept shredding pirates because they all have quad blaster turrets. I'd definitely like to see more Haulers in the FW storyline because most of the time we're using Freighters or Star Barges, while in FW space I barely see any civilian Freighters.

Reply:
The composition of FW fleets is something we can probably work on later.